src/mixins/filing-template-mixin.ts

```diff
@@ -283,6 +283,29 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     // Include name request info when applicable
     if (this.hasNewNr || this.hasBusinessNameChanged) filing.alteration.nameRequest = { ...this.getNameRequest }
 
+    // Populate Staff Payment according to payment option
```

Comment:
This looks identical to the code for corrections... could this use a new helper method?

Reply:
It is! Yes you bet, i did the same for the parse, should have done this for the build 😁

Reply:
Updated